e2e/src/helper.ts

```diff
@@ -150,6 +150,13 @@ export async function compileDocument(): Promise<void> {
   await commands.executeCommand('WizardForDbtCore(TM).compile');
 }
 
+export async function acceptFirstSuggestion(): Promise<void> {
+  await commands.executeCommand('editor.action.triggerSuggest');
+  await sleep(400);
+  await commands.executeCommand('acceptSelectedSuggestion');
+  await sleep(300);
+}
```

Comment:
If we can use the `ExtensionApi` with smaller timeouts?
```
completionProvider: {
          resolveProvider: true,
```
Or if we can listen to text changes?

Reply:
I got rid of the last sleep, thanks for suggestion. I'm not sure if we can get rid of first sleep since we don't control VS Code `editor.action.triggerSuggest` command.